WordPress/src/main/res/values/strings.xml

```diff
@@ -1621,7 +1621,7 @@
     <string name="alert_error_adding_media">An error occurred while inserting media</string>
 
     <string name="stop_upload_dialog_title">Stop uploading?</string>
-    <string name="stop_upload_button">Stop Upload</string>
+    <string name="stop_upload_button" translatable="false">@string/ok</string>
```

Comment:
We could do that now since we're just going with `OK`. I'll remove the string resource from the editor project as well as WPAndroid.